frontend/src/environments/environment.prod.ts

```diff
@@ -2,4 +2,5 @@ export const environment = {
     production: true,
     coreUrl: '/api',
     notifierUrl: '/notifier',
+    googleClientId: 'TBD',
```

Comment:
what is this?

Reply:
this is `googleClientId` that we will have for Prod env (TBD means `to be defined` later)